create_person.rb

```diff
@@ -0,0 +1,36 @@
+class CreatePerson
+  def initialize(state)
+    @state = state
+  end
+
+  def create_person
+    print 'Do you want to create a student (1)  or a teacher (2)? [Input the number]: '
+    user_choice = gets.chomp
+    print 'Age: '
+    @age = gets.chomp.to_i
+    print 'Name: '
+    @name = gets.chomp
```

Comment:
- [ ] Kindly ensure that requests for user inputs are handled by another method in keeping with the single-responsibility rule.